src/modules/party/dto/create-party-request.dto.ts

```diff
@@ -0,0 +1,52 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsIn, IsISO8601, IsOptional, IsString, Length, Matches } from 'class-validator';
+
+export type CreatePartyRequest = CreatePartyRequestItem[];
+
+export class CreatePartyRequestItem {
+  @ApiProperty({ description: 'The UKEF ID for the party. Should contain just digits.', minLength: 8, maxLength: 8, example: '00291013' })
+  @Length(8, 8, {
+    message: 'alternateIdentifier must be exactly 8 characters',
+  })
+  @Matches(/^\d+$/, {
+    message: 'alternateIdentifier must only contain digits',
+  })
+  alternateIdentifier: string;
+
+  @ApiProperty({ minLength: 1, maxLength: 10, example: '0001' })
+  @Length(1, 10)
+  industryClassification: string;
+
+  @ApiProperty({ minLength: 1, maxLength: 35, example: 'ACTUAL IMPORT EXPORT' })
+  @Length(1, 35)
+  name1: string;
+
+  @ApiProperty({ required: false, minLength: 1, maxLength: 35, example: 'ACTUAL IMPORT EXPORT' })
+  @IsOptional()
+  @Length(0, 35)
+  name2: string;
+
+  @ApiProperty({ required: false, minLength: 1, maxLength: 35, example: 'ACTUAL IMPORT EXPORT' })
+  @IsOptional()
+  @Length(0, 35)
+  name3: string;
+
+  @ApiProperty({ minLength: 1, maxLength: 2, example: '70' })
+  @Length(1, 2)
+  smeType: string;
+
+  @ApiProperty({ description: "Should be '1' if the domicile country is the UK, otherwise '2'.", example: '2' })
+  @IsString()
+  @IsIn(['1', '2'])
+  citizenshipClass: string;
+
+  @ApiProperty({ description: 'The date of creation in YYYY-MM-DD (ISO 8601) format.', example: '2023-03-15' })
+  @IsISO8601()
+  @Length(0, 10)
+  officerRiskDate: string;
+
+  @ApiProperty({ description: "Country code for the party's primary address.", required: false, maxLength: 3, default: 'GBR', example: 'DZA' })
```

Comment:
Can we make sure all the descriptions are full sentences please? E.g. "The country code for the party's primary address."